src/components/Footer/Footer.scss

```diff
@@ -7,7 +12,12 @@
 
   @media (min-width: 1400px) {
     max-width: 1200px;
-    margin: 0 auto;
+    margin: 0 auto;;
```

Comment:
It`s a kind of double protection? 😄  or what is the reason for the second semicolon?)